apps/aecore/src/aec_peer_connection.erl

```diff
@@ -957,7 +957,7 @@ handle_new_micro_block(S, Msg) ->
                 %% in the full micro block case - conductor will do the necessary checks
```

Comment:
In the case of full micro block as well as in case of light micro block, conductor performs checks though in singleton process that may get overloaded if e.g. PoF in microblock has two headers that are the same. We may consider moving some validation outside of aec_conductor? Though optimization so not in scope for this PR. Ok.

Reply:
Re this point in scope of PoF: there are two signature verifications more that the conductor will run for PoF, and I am not sure if those verification has negligible time. Hence the previous comment.

Reply:
I see this already tracked as https://www.pivotaltracker.com/story/show/153642409